parm/config/config.fcst

```diff
@@ -276,6 +349,7 @@ fi
 export REMAPSH="$HOMEgfs/ush/fv3gfs_remap.sh"
 export master_grid="0p25deg"                   # 1deg 0p5deg 0p25deg 0p125deg etc
 export npe_remap=$((npe_fcst < 240 ? npe_fcst : 240))
+export nth_remap=2
```

Comment:
Is anyone useing `regrid_nemsio` and `nc2nemsio` anymore?

Reply:
I had a user contact me recently about fv3nc2nemsio but that's the only person I know for sure is using one of those. I'll remove nth_remap=2 to bring it back to how config.fcst is in develop, that variable should be in config.resources if needed anyway.

Reply:
We don't use nemsio any more, so I don't see a reason to maintain it. If others want it, they need to either create their own or use an old version of the workflow.

Reply:
That's not to say this needs to be addressed now. We can remove it later.

Reply:
Removing remap and regrid blocks in config.fcst.